spec/features/admin/order_cycles_spec.rb

```diff
@@ -181,11 +181,17 @@
       select2_select 'My coordinator', from: 'coordinator_id'
       click_button "Continue >"
 
-      # And I fill in the basic fields
+      # I cannot save before filling in the required fields
+      expect(page).to have_button("Create", disabled: true)
+
+      # After I fill in the basic fields
       fill_in 'order_cycle_name', with: 'Plums & Avos'
       fill_in 'order_cycle_orders_open_at', with: order_cycle_opening_time
       fill_in 'order_cycle_orders_close_at', with: order_cycle_closing_time
 
+      # I can save the form
+      expect(page).to have_button("Create", disabled: false)
```

Comment:
maybe we can add a comment here about the existing incoherence between the button showing and the actual mandatory fields? as far as I understand the button is showing even if the mandatory fields pickup_time and pickup_instructions are not filled in.